archive-cleaner

```diff
@@ -68,32 +67,48 @@ def get_packages_from_buildinfo(archive, filename, archive_entry, arch):
     packages = []
     if buildinfo_format > 0:
         for package in installed:
-            pkgfilename = f'{package}.pkg.tar.xz'
+            pkgfilenames = [f'{package}.pkg{ext}' for ext in PKG_EXTENSIONS]
             pkgname = '-'.join(package.split('-')[:-3])
             if not pkgname:
                 print(f'ERROR: failed to parse {package} with format {buildinfo_format} needed for {filename}', file=stderr)
                 continue
-            directory = join(archive, 'packages', pkgname[0], pkgname)
-            path = join(directory, pkgfilename)
-            if not isfile(path):
-                print(f'ERROR: {package} needed for {filename} not in archive {path}', file=stderr)
+            directory = join(archive, 'packages'. pkgname[0], pkgname)
+            paths = [(pkgfilename, join(directory, pkgfilename)) for pkgfilename in pkgfilenames]
+            existing_paths = list(filter(lambda path_tuple: isfile(path_tuple[1]), paths))
```

Comment:
it shouldn't be possible to get the same artifact release with multiple extensions, lets make the code lot easier by checking and extracting just a single valid path to continue the work with

Reply:
This is something I thought and wasn't sure about it, will change it